src/drawing/conics.rs

```diff
@@ -211,11 +211,12 @@ where
         draw_if_in_bounds(image, x0 + y, y0 - x, color);
         draw_if_in_bounds(image, x0 + x, y0 - y, color);
 
-        y += 1;
-        err += 1 + 2 * y;
-        if 2 * (err - x) + 1 > 0 {
-            x -= 1;
-            err += 1 - 2 * x;
+        x += 1;
+        if (p < 0) {
+            p += 2*x + 1;
```

Comment:
Nit: add a space on both sides of all arithmetic operators here and below.

Reply:
Fixed.